app/ui-react/syndesis/src/modules/integrations/components/editor/choice/utils.ts

```diff
@@ -20,3 +20,35 @@ export function createChoiceConfiguration(configuredProperties: {
     routingScheme,
   } as IChoiceConfiguration;
 }
+
+/**
+ * Builds a sane flow form option description from either condition expression or path, op, value properties
+ * @param option
+ */
+export function getFlowDescription(option: IFlowFormOption) {
+  if (option.condition) {
+    return option.condition;
+  }
+
+  if (option.path) {
+    return '${body.' + option.path + '} ' + option.op + " '" + option.value + "'";
+  }
+
+  return "";
+}
+
+/**
+ * Builds a sane flow option condition expression from either given condition expression or path, op, value properties
+ * @param option
+ */
+export function getConditionExpression(option: IFlowOption) {
+  if (option.condition) {
+    return option.condition;
+  }
+
+  if (option.path) {
+    return '${body.' + option.path + '} ' + option.op + " '" + option.value + "'";
```

Comment:
Same comment as above, I think an interpolated string might be more clear and less quote-y.  Also if this function and the last function do the same thing, why have two of them?